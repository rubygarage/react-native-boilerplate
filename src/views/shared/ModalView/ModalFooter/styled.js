import styled from 'styled-components/native';

const Wrap = styled.View`
  flex-shrink: 0;
  flex-direction: row;
  justify-content: flex-end;
  padding: 16px;
`;

// eslint-disable-next-line import/prefer-default-export
export { Wrap };
