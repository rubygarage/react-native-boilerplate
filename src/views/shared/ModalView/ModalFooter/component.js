import React from 'react';
import PropTypes from 'prop-types';

import * as S from './styled';

const ModalFooter = ({ children }) => (
  <S.Wrap>
    {children}
  </S.Wrap>
);

ModalFooter.propTypes = {
  children: PropTypes.node.isRequired,
};

export default ModalFooter;
