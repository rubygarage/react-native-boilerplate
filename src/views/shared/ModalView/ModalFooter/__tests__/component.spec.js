import React from 'react';
import { View } from 'react-native';
import { shallow } from 'enzyme';

import ModalFooter from '../component';

const component = shallow(<ModalFooter><View /></ModalFooter>);

describe('ModalFooter component', () => {
  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
