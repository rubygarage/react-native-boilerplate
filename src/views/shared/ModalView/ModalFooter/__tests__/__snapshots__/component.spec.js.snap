// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter component renders correctly 1`] = `
<Styled(View)>
  <View />
</Styled(View)>
`;
